extension/src/popup/helpers/fetch.ts

```diff
@@ -0,0 +1,14 @@
+export const fetchJson = async <T>(url: string, options?: RequestInit) => {
```

Comment:
We can probably use this everywhere that we use `fetch` now but I wanted to get this out first to make sure it makes to the upcoming patch.